mlrun/serving/states.py

```diff
@@ -1735,12 +1736,29 @@ def _get_model_output_schema(
             return [feature.name for feature in _model_artifact.spec.outputs]
 
     @staticmethod
-    def _get_model_endpoint_output_schema(
+    def _get_model_input_schema(
+        model_artifact: Union[ModelArtifact, LLMPromptArtifact],
+    ) -> Optional[list[str]]:
+        if isinstance(
+            model_artifact,
+            ModelArtifact,
+        ):
+            return [feature.name for feature in model_artifact.spec.inputs]
+        elif isinstance(
+            model_artifact,
+            LLMPromptArtifact,
+        ):
+            _model_artifact = model_artifact.model_artifact
```

Comment:
This line can cause API call.. can we some how do it once for outputs and inputs?